src/Symfony/Component/HttpFoundation/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ CHANGELOG
    according to [RFC 8674](https://tools.ietf.org/html/rfc8674)
  * made the Mime component an optional dependency
  * added `MarshallingSessionHandler`, `IdentityMarshaller`
+ * made `Session` accept a callback reporting when the session is being used
```

Comment:
s/reporting/to report actually, sorry for the not ideal initial suggestion

Reply:
No worries, I've changed it :)